gnovm/pkg/gnolang/uverse.go

```diff
@@ -95,23 +95,23 @@ func UverseNode() *PackageNode {
 	def("._", undefined)   // special, path is zero.
 	def("iota", undefined) // special
 	def("nil", undefined)
+	def("bigint", asValue(BigintType))
 	def("bool", asValue(BoolType))
-	def("string", asValue(StringType))
+	def("byte", asValue(Uint8Type))
+	def("float32", asValue(Float32Type))
+	def("float64", asValue(Float64Type))
 	def("int", asValue(IntType))
 	def("int8", asValue(Int8Type))
 	def("int16", asValue(Int16Type))
-	def("rune", asValue(Int32Type))
 	def("int32", asValue(Int32Type))
 	def("int64", asValue(Int64Type))
+	def("rune", asValue(Int32Type))
```

Comment:
rune belongs next to int16 since it is an alias?

Reply:
they're sorted alphabetically. do you prefer that I group them by underlying type instead?